rfcs/text/0008-threat-intel.md

```diff
@@ -87,171 +85,139 @@ Stage 3: Add or update all remaining field definitions. The list should now be e
-        "pid": 1644,
-        "entity_id": "MDgyOWFiYTYtMzRkYi1kZTM2LTFkNDItMzBlYWM3NDVlOTgwLTE2NDQtMTMyNDk3MTA2OTcuNDc1OTExNTAw",
-        "executable": "C:\\Windows\\System32\\svchost.exe"
-    },
-    "message": "Endpoint file event",
-    "@timestamp": "2020-11-17T19:07:46.0956672Z",
-    "file": {
-        "path": "C:\\Windows\\Prefetch\\SVCHOST.EXE-AE7DB802.pf",
-        "extension": "pf",
-        "name": "SVCHOST.EXE-AE7DB802.pf",
-        "hash": {
-            "sha256": "0c415dd718e3b3728707d579cf8214f54c2942e964975a5f925e0b82fea644b4"
-        }
-    },
-    "threat": {
-        "indicator": [
-            {
-                // Each enrichment is added as a nested object under `threat.indicator.*`
-                // Copy all the object indicators under `indicator.*`, providing full context
-                "file": {
-                    "hash": {
-                        "sha256": "0c415dd718e3b3728707d579cf8214f54c2942e964975a5f925e0b82fea644b4",
-                        "md5": "1eee2bf3f56d8abed72da2bc523e7431"
-                    },
-                    "size": 656896,
-                    "name": "invoice.doc"
-                },
-                /* `matched` will provide context about which of the indicators above matched on this
-                    particular enrichment. If multiple matches for this indicator object, this could
-                    be a list */
-                "matched": "sha256",
-                "marking": {
-                    "tlp": "WHITE"
-                },
-                "first_seen": "2020-10-01",
-                "last_seen": "2020-11-01",
-                "sightings": 4,
-                "type": ["sha256", "md5", "file_name", "file_size"],
-                "description": "file last associated with delivering Angler EK",
-
-                // Copy event.* data from source threatintel document
-                "provider": "Abuse.ch",
-                "dataset": "threatintel.abusemalware",
-                "module": "threatintel"
-            }
-        ]
+    // Metadata about the indicator event
+    "@timestamp": "2019-08-10T11:09:23.000Z",
+    "event": {
+        "kind": "enrichment",
+        "category": "threat",
+        "type": "indicator",
+        "reference": "https://bazaar.abuse.ch",
+        "severity": 7,
+        "risk_score": 10,
+        "original": "2020-10-29 19:16:38"
+        },
+
+    // Metadata about the indicator data
+    "threat.indicator": {
+        "first_seen": "2020-10-01",
+        "last_seen": "2020-11-01",
+        "sightings": "10",
+        "type": [
+            "file"
+        ],
+        "description": "Implant used during an Angler EK campaign.",
+        "dataset": "threatintel.malwarebazaar",
+        "module": "threatintel",
+        "provider": "Abuse.ch",
+        "reference": "https://bazaar.abuse.ch/sample/f3ec9a2f2766c6bcf8c2894a9927c227649249ac146aabfe8d26b259be7d7055",
+        "confidence": "High",
+        "file": {
```

Comment:
Apologies, I know I keep circling back to this topic, but I also keep seeing instances of it:

For the indicator use case, would this actually be the top-level `file.*` fields, or nested under `threat.indicator.*` like the example here?

@rylnd @peasead

Reply:
@ebeahan I agree, I think several of these examples should be moved up a level, e.g.:

* `threat.indicator.dataset` -> `event.dataset`
* `threat.indicator.reference` -> `event.reference`
* `threat.indicator.file.*` -> `file.*`

Reply:
Thanks, @rylnd and @ebeahan 

After reviewing the [threat enrichment RFC](https://github.com/rylnd/ecs/blob/threat-enrichment-stage-1/rfcs/text/0021-threat-enrichment.md), I understand better. 

I'll make those changes.

Reply:
@rylnd and @ebeahan 

Shouldn't we move `threat.indicator.domain` to the existing ECS URL fieldset (`url.domain`) as well?

Reply:
Going with `.domain` being under the existing URL fieldset (`url.domain.`).